docs/design/diagram-types/DesignDiagramsTemplates/componentDiagrams.md

```diff
@@ -0,0 +1,76 @@
+# Component Diagrams
+
+## Purpose
+
+This document is intended to provide a baseline understanding for what, why, and how to incorporate Component Diagrams
+as part of your engagement. Regarding the **how**, the section at the bottom will provide tools and plugins to streamline as much as possible when generating Component Diagrams through VSCode.
+
+[Wikipedia](https://en.wikipedia.org/wiki/Component_diagram) defines UML Component Diagrams as:
+
+ > _a component diagram depicts how components are wired together to form larger components or software systems._
+
+Component Diagrams are a type of a static structure because it focuses on the responsibility and relationships between components as part of the overall system or solution.
+
+It is not supposed to inform about the data flow, the caller or callee responsibilities, the request flows, nor any other "behavior" related characteristics.
+
+...Hold on a second... what is a Component?
+
+A [Component](https://en.wikipedia.org/wiki/Component_(UML)) is a runnable solution that performs a set of operations and can possibly be interfaced through a particular API. One can think of Components as a "stand alone" piece of software - think datastores, microservices, serverless functions, user interfaces, etc...
+
+## Essential Takeaways
+
+The primary two takeaways from a Component Diagram should be:
+
+1. A quick view of all the various components (User Interface, Service, Data Storage) involved in the system
```

Comment:
Are these meant to be ordered? if so, please ignore this, otherwise use an unordered lists

Reply:
Changed them to an unordered list - thank you!